LoopProjectFile/ExtractedInformation.py

```diff
@@ -302,26 +303,55 @@ def SetStratigraphicLog(root, data, append=False, verbose=False):
             zlib=True,
             complevel=9,
         )
+        
+        # Define compound type for thickness calculator (list of length 5)
+        thicknessCalculatorType_t = siGroup.createCompoundType(
+            LoopProjectFile.thicknessCalculatorType, "ThicknessCalculator"
+        )
+        siGroup.createVariable(
+            "thicknessCalculator",
+            thicknessCalculatorType_t,
+            zlib=True,
+            complevel=9,
+        )
     else:
         siGroup = resp["value"]
 
     if siGroup:
         stratigraphicLayersLocation = siGroup.variables["stratigraphicLayers"]
+        thickness_calculator = siGroup.variables["thicknessCalculator"]
+        
         index = 0
         if append:
             index = siGroup.dimensions["index"].size
         for i in data:
             stratigraphicLayersLocation[index] = i
             index += 1
-        siGroup.setncattr("index_MaxValid", index)
+         # Write thickness calculator data (must match expected structure)
```

Comment:
This is good, but I'm not sure how LoopStructural can know which thickness to use. Maybe we need to have another column "activeThickness"

Reply:
Added